src/lightstep-graph/module.js

```diff
@@ -27,7 +27,7 @@ class GraphCtrl extends MetricsPanelCtrl {
     this.annotations = [];
     this.alertState = null;
 
-    this.annotationsPromise = null;
+    this.annotationsPromise = Promise.resolve({ annotations: [] });
```

Comment:
Setting a promise as an initial value lets us render on first [pass](https://github.com/lightstep/lightstep-grafana-plugin/blob/master/src/lightstep-graph/module.js#L212).

Grafana's native panel implementations served as a good point of reference for this [patch](    this.annotationsPromise = Promise.resolve({ annotations: [] });).